pkg/manager/manager.go

```diff
@@ -155,6 +144,32 @@ func (m *SupportBundleManager) Run() error {
 	select {}
 }
 
+func (m *SupportBundleManager) initClients() error {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return err
+	}
+	m.restConfig = config
+	hvst, err := client.NewHarvesterClient(m.context, m.HarvesterNamespace, m.restConfig)
+	if err != nil {
+		return err
+	}
+	m.harvester = hvst
+
+	k8s, err := client.NewKubernetesClient(m.context, m.HarvesterNamespace, m.restConfig)
+	if err != nil {
+		return err
+	}
+	m.k8s = k8s
+
+	k8sMetrics, err := client.NewMetricsClient(m.context, m.HarvesterNamespace, m.restConfig)
+	if err != nil {
+		return err
+	}
+	m.k8sMetrics = k8sMetrics
+	return nil
+}
```

Comment:
u should be able to do the assignment directly instead of after the error check as it will return the error if it has one.

Reply:
Thanks, [fixed](https://github.com/harvester/support-bundle-utils/pull/4/commits/f533e4fcd2da6c261974ed957b28b077c0f05795).